_RFCs/0005-rfc-java-semantic-versioning.md

```diff
@@ -0,0 +1,46 @@
+- Change Name: java-semantic-versioning
+- Start Date: 2018-03-29
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes that supported java versions are adapted to semantic versioning `M.m.p` as the rest of habitat packages
+
+# Motivation
+[motivation]: #motivation
+
+Habitat has been designed to work with semantic versions of the format `M.m.p`, and package newness is based on this format. Currently, new versions of the java packages have to be pinned explicitely or else, you might never get those as dependency since, for example, 8u92 is found newer than 8u151.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Java packages (jre and jdk) with version <= 8 will now adapt the official java version to habitat's package system:
+
+- core/jre8/8u151 will become core/jre8/8.0.151
+- core/jdk8/8u92 will become core/jdk8/8.0.92
+
+Generally, versions `<M>u<p>` will be renamed `<M>.0.<p>`
```

Comment:
I'd recommend using `major.minor.0` so these would become `core/jdk8/8.92.0` and `core/jre8/8.151.0`. Rationale here being that these are minor releases to a major version and not patch releases.

Reply:
Actually, it depends on the update of the JDK, but that might indeed be better ;) thanks!